src/components/graph/menus/network-modifications-menu.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import Menu from '@mui/material/Menu';
+import { useIntl } from 'react-intl';
+import { NestedMenuItem } from 'mui-nested-menu';
+import NodeMenuItem from './create-node-item';
+import MenuList from '@mui/material/MenuList';
+
+/**
+ * Menu to select network modification to create
+ * @param {Boolean} open Is the dialog open ?
+ * @param {EventListener} onClose Event to close the dialog
+ * @param onOpenDialog handle the opening of dialogs
+ * @param dialogs the list of dialog
+ * @param anchorEl anchorEl of fab Button
+ */
+const NetworkModificationsMenu = ({
+    open,
+    onClose,
+    onOpenDialog,
+    dialogs,
+    anchorEl,
+}) => {
+    const intl = useIntl();
+    const renderMenuItems = (menuDialogs) => {
+        return menuDialogs.map((dialog) => {
+            return (
+                <MenuList key={dialog.id}>
+                    {dialog.subItems === undefined ? (
+                        <NodeMenuItem
```

Comment:
Where does this NodeMenuItem comes from ?
from the contextual menu in tree graph ? it's not the same context here, it's confusing. Choose between making a generic XXXMenuItem component which fits for multiple usage, or creating a new component here. There should be no reason to depends of this component evolution in here.

Reply:
done